scripts/reflow.py

```diff
@@ -113,10 +113,10 @@
 # A single letter followed by a period, typically a middle initial.
 endInitial = re.compile(r'^[A-Z]\.$')
 # An abbreviation, which doesn't (usually) end a line.
-endAbbrev = re.compile(r'(e\.g|i\.e|c\.f)\.$', re.IGNORECASE)
+endAbbrev = re.compile(r'(e\.g|i\.e|c\.f|vs)\.$', re.IGNORECASE)
```

Comment:
@oddhack BTW, first time I heard of "cf.". Doesn't see, to be used in the spec. Per internet, "c.f." seems to be incorrect spelling